### PR TITLE
Add map section

### PR DESCRIPTION
**Pull request by nd0761**

### CODE REVIEW

File: src/main/webapp/.prettierignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Ignore artifacts:
+build
+coverage
```
Comment (ioannanedelcu) [inline src/main/webapp/.prettierignore:3]: I am not sure if this file should be included here

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -44,19 +53,21 @@ <h1>TheGlobetrotter</h1>
...
-          <span class="fas fa-ellipsis-v"></span>Place 3</div>
-        <div class="new-place" id="place-id-4">
-          <span class="fas fa-ellipsis-v"></span>Place 4</div>
-        <div class="new-place" id="place-id-5">
-          <span class="fas fa-ellipsis-v"></span>Place 5</div>
-        <div class="new-place" id="place-id-6">
-          <span class="fas fa-ellipsis-v"></span>Place 6</div>
+        <div id="route-creation-name"> <p>Route name</p> </div>
+        <ul id="places-table">
+          <li class="new-place" id="place-id-1">
```
Comment (ioannanedelcu) [inline src/main/webapp/index.html:83]: Could you make this table bigger so that it would be more visible for the users?

Comment (nd0761) [reply src/main/webapp/index.html:83]: Increased it to 500px and change height to auto.

File: src/main/webapp/.prettierignore (hunk 1 above)
Comment (nd0761) [reply src/main/webapp/.prettierignore]: I'm not sure if it does anything, so yeah, deleted it.

File: src/main/webapp/index.html (hunk 2 above)
Comment (ioannanedelcu) [reply src/main/webapp/index.html:83]: Great, thanks

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -1,3 +1,26 @@
+<!--
+  Copyright OpenJS Foundation and other contributors, https://openjsf.org/
```
[the remark this answers is not in the document]
Comment (nd0761) [reply src/main/webapp/index.html:2]: Oooh, thank you!

**Comment by nd0761** [comment]
I tried but yeah, It still won't check(

**Comment by conradgrobler** [comment]
> I tried but yeah, It still doesn't want to check(

That is weird. Perhaps try to push an update. One option is to just update the timestamp on the last commit by using `git commit --amend` and then push the change using `git push origin mapSection --force`.

**Comment by nd0761** [comment]
It worked!! )

File: src/main/webapp/mapScript.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+let map;
+
+function initMap(showMarkers = true) {
+  // Create markers.
+  let japanTemple = { lat: 34.462117, lng: 135.830272 };
+  let newZealandLake = { lat: -43.979931, lng: 170.194799 };
+  let ugandaView = { lat: -0.099273, lng: 32.652921 };
```
Comment (ipetr0v) [inline src/main/webapp/mapScript.js:21]: I think it's better to have [default parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) than having hardcoded some coordinates.
Or you can pass these parameters as normal ones.

File: src/main/webapp/script.js [hunk 5]
```diff
@@ -11,3 +11,18 @@
...
+
+// Make navbar stick to the top of the page when scrolling.
+window.onscroll = function () {
+  stickyNavbar();
+};
+
+function stickyNavbar() {
+  var navbar = document.getElementById("navbar");
+
+  if (window.pageYOffset >= 308) {
```
Comment (ipetr0v) [inline src/main/webapp/script.js:23]: I think it's better to have named constants (possibly global) for such things

File: src/main/webapp/mapScript.js (hunk 4 above)
Comment (nd0761) [reply src/main/webapp/mapScript.js:21]: Changed them to default.

**Pull request merged**